go.mod

```diff
@@ -3,7 +3,7 @@ module chainguard.dev/melange
 go 1.24.0
 
 require (
-	chainguard.dev/apko v0.27.9
+	chainguard.dev/apko v0.27.10-0.20250623150342-bfcb016b5959
```

Comment:
Out of curiosity, is this correct?  I'm not sure whether we should release apko first and then depend on the released version here, or if it's OK to just depend on unreleased commits.  Genuinely curious!

Reply:
That makes sense, thanks!